src/resource/tiles/mock-tile.ts

```diff
@@ -0,0 +1,10 @@
+export const mockTile = {
+  name: "Elite X II",
+  colors: {
+    red: "FF0000",
+  },
+  length: 300,
+  width: 300,
+  height: 15,
+  _id: "1",
+};
```

Comment:
Eason, caz you put colors as an object, why not put length, width and height together as an object?

Reply:
If put into an object, it is really hard to do validation.